scripts/sct_deepseg.py

```diff
@@ -130,34 +145,82 @@ def main():
                 default[True]))
         exit(0)
 
+    # Deal with task
+    if args['list_tasks']:
```

Comment:
looks a lot like the code for list-models-- we should probably make it a function